src/Hedgehog/Tuple.fs

```diff
@@ -0,0 +1,19 @@
+namespace Hedgehog
+
+module private Tuple =
+
+    let mapFirst (f : 'a -> 'c) (x : 'a, y : 'b) : 'c * 'b =
+        f x, y
+
+    let mapSecond (f : 'b -> 'c) (x : 'a, y : 'b) : 'a * 'c =
+        x, f y
+
+
+module private GenTuple =
```

Comment:
As I said in https://github.com/hedgehogqa/fsharp-hedgehog/pull/260#discussion_r551046128, I think a better name for this module is `GenPair`.

Reply:
I'm going to leave these as is, I don't think they hurt anything.

Reply:
The PR has already been completed, so this is a bit late.  Nonetheless, another point of comparison is with [FSharpPlus, which calls this `Tuple2`.](https://github.com/fsprojects/FSharpPlus/blob/a36d35a3470a0a59b176e7527f426f9014333c1d/src/FSharpPlus/Extensions/Tuple.fs)

Reply:
There's still room for changes, if you want. Until we release the changes. (And even after that, theoretically, we can rename those, as per SemVer.)